### ISSUE TITLE
Unknown option libnkutils:uuid

### ISSUE THREAD

**Issue by jubalh** [DaveDavenport/rofi#677]
Compiling with meson and disabled check dependency.

## Version

1.4.1

## Steps to reproduce
* grab tarball
* run: `meson build -Denable-check=false`

## What behaviour you see
```
meson build -Denable-check=false                                                                                                                      
The Meson build system
Version: 0.42.0
Source dir: /home/michael/src/rofi-1.4.1
Build dir: /home/michael/src/rofi-1.4.1/build
Build type: native build
Project name: rofi
Native C compiler: cc (gcc 7.2.1)
Build machine cpu family: x86_64
Build machine cpu: x86_64
Compiler for C supports argument -Wparentheses: YES
Compiler for C supports argument -Winline: YES
Compiler for C supports argument -Wunreachable-code: YES
Compiler for C supports argument -Werror=missing-prototypes: YES
Found pkg-config: /usr/bin/pkg-config (0.29.2)
Native dependency glib-2.0 found: YES 2.54.0
Native dependency gio-unix-2.0 found: YES 2.54.0
Native dependency gmodule-2.0 found: YES 2.54.0
Native dependency cairo found: YES 1.15.8
Native dependency pango found: YES 1.40.12
Native dependency pangocairo found: YES 1.40.12
Native dependency xkbcommon found: YES 0.7.2
Native dependency librsvg-2.0 found: YES 2.40.18
Library m found: YES

Executing subproject libgwater/xcb.

Project name: libgwater-xcb
Native C compiler: cc (gcc 7.2.1)
Native dependency xcb found: YES 1.12
Build targets in project: 1

Subproject libgwater/xcb finished.
Native dependency xcb-aux found: YES 0.4.0
Native dependency xcb-xkb found: YES 1.12
Native dependency xkbcommon-x11 found: YES 0.7.2
Native dependency xcb-ewmh found: YES 0.4.1
Native dependency xcb-icccm found: YES 0.4.1
Native dependency xcb-xrm found: YES 1.2
Native dependency xcb-randr found: YES 1.12
Native dependency xcb-xinerama found: YES 1.12
Native dependency cairo-xcb found: YES 1.15.8
Native dependency libstartup-notification-1.0 found: YES 0.12
Dependency check found: NO
Program git found: YES (/usr/bin/git)
Configuring config.h using configuration

Executing subproject libnkutils.

Project name: libnkutils
Native C compiler: cc (gcc 7.2.1)

Meson encountered an error in file subprojects/libnkutils/meson.build, line 60, column 7:
Tried to access unknown option "libnkutils:uuid".
```

## What behaviour you expect to see

* It compiles

**Comment by DaveDavenport** [comment]
We will fix it for 1.4.2, please for now use  autotools build system.

**Comment by sardemff7** [comment]
Fixed in b7a36f5ea50649b514c54591530e1bafc15b3578
